dual-tokenomics/dual-dao/README.md

```diff
@@ -4,15 +4,12 @@ description: Governance of Dual Finance
 
 # Dual DAO
 
-Deposit DUAL to vote on Dual DAO proposals on [Realms](https://app.realms.today/dao/dual%20dao)\
+Dual DAO is hosted on [Realms](https://app.realms.today/dao/dual%20dao).\
 \
-Treasury earns fees on DIPs and exercised SOs.\
+The DAO enables participants to deposit DUAL tokens in return for elibility to vote on governace proposals. There is no minimum deposit size and there is no lock-up period for the deposit; users can withdraw their tokens at any time.\
 \
```

Comment:
Add screenshot of governance parameters: 
![image](https://user-images.githubusercontent.com/107803907/227381770-a2648d84-6fdd-4a3a-a9f7-a1f4e60514d4.png)

Reply:
Added.